src/otx/algo/common/utils/utils.py

```diff
@@ -259,3 +260,69 @@ def inverse_sigmoid(x: Tensor, eps: float = 1e-5) -> Tensor:
     x1 = x.clamp(min=eps)
     x2 = (1 - x).clamp(min=eps)
     return torch.log(x1 / x2)
+
+
+def cut_mixer(images: Tensor, masks: Tensor) -> tuple[Tensor, Tensor]:
```

Comment:
It seems to be used just for semantic segmentation, how about moving into semantic segmentation?

Reply:
It will be actually used for other tasks with Semi-SL based on the MeanTeacher approach that we will add in the future. (Object Detection and Instance Segmentation)

Reply:
However, you are right that it is related to segmentation mostly due to operations with masks. Since I started refactoring task, can I move it in that PR? I would like to merge this PR since my changes depends on it